src/AsyncJson.h

```diff
@@ -0,0 +1,76 @@
+// ESPasyncJson.h
+/*
+  Async Response to use with arduinoJson and asyncwebserver
+  Written by Andrew Melvin (SticilFace) with help from me-no-dev and BBlanchon.
+
+  example of callback in use
+
+   server.on("/json", HTTP_ANY, [](AsyncWebServerRequest * request) {
+
+    AsyncJsonResponse * response = new AsyncJsonResponse();
+    JsonObject& root = response->getRoot(); 
+    root["key1"] = "key number one";
+    JsonObject& nested = root.createNestedObject("nested"); 
+    nested["key1"] = "key number one";
+
+    response->setLength();
+    request->send(response); 
+  });
+
+*/
+
+#pragma once
+#include <ArduinoJson.h> 
+
+/*
+ * Json Response
+ * */
+
+class ChunkPrint : public Print{
+  private:
+    uint8_t* _destination;
+    size_t _to_skip;
+    size_t _to_write;
+    size_t _pos;
+  public:
+    ChunkPrint(uint8_t* destination, size_t from, size_t len)
+      : _destination(destination), _to_skip(from), _to_write(len), _pos{0} {}
+    size_t write(uint8_t c){
+      if (_to_skip > 0) {
+        _to_skip--;
+        return 0; 
```

Comment:
I wander if it should return 1 here? If ArduinoJson would have stopped sending when it returns 0 then time would have been less. Maybe we should look at that and fix it there :)